Error: The `form` element was not requested.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
diff --git a/output/Radio Group/1/e18596ae-8c40-4aca-94b0-1c424a25ad0d.html b/output/Radio Group/1/e18596ae-8c40-4aca-94b0-1c424a25ad0d.html
index 27702c6..aac94c5 100644
--- a/output/Radio Group/1/e18596ae-8c40-4aca-94b0-1c424a25ad0d.html	
+++ b/output/Radio Group/1/e18596ae-8c40-4aca-94b0-1c424a25ad0d.html	
@@ -1,17 +1,15 @@
-<form>
-  <fieldset>
-    <legend>Choose a theme:</legend>
-    <label>
-      <input type="radio" name="theme" value="light">
-      Light
-    </label>
-    <label>
-      <input type="radio" name="theme" value="dark">
-      Dark
-    </label>
-    <label>
-      <input type="radio" name="theme" value="high-contrast">
-      High Contrast
-    </label>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset>
+  <legend>Choose a theme:</legend>
+  <label for="light">
+    <input type="radio" name="theme" id="light" value="light">
+    Light
+  </label>
+  <label for="dark">
+    <input type="radio" name="theme" id="dark" value="dark">
+    Dark
+  </label>
+  <label for="high-contrast">
+    <input type="radio" name="theme" id="high-contrast" value="high-contrast">
+    High Contrast
+  </label>
+</fieldset>
\ No newline at end of file
